### PR TITLE
DEVTOOLS-95: Save audit information for root RAML in create.

### PR DESCRIPTION
**Pull request by juancoen**

### CODE REVIEW

File: tests/unit/apiPlatformService.test.js [hunk 1]
```diff
@@ -539,6 +551,8 @@ function checkCreateAPICalls(api) {
   loggerStub.info.secondCall.calledWithExactly(uploadingRootRaml);
   loggerStub.info.thirdCall.calledWithExactly(rootRamlUploaded);
 
+  console.log(api);
+  console.log(newApi);
```
Comment (fschargorosdky) [inline tests/unit/apiPlatformService.test.js]: debbuging code

File: tests/unit/apiPlatformService.test.js [hunk 2]
```diff
@@ -88,7 +100,7 @@ describe('apiPlatformService', function () {
       apiPlatformRepositoryStub.createAPI =
         sinon.stub().returns(BPromise.resolve(newApi));
       apiPlatformRepositoryStub.addRootRaml =
-        sinon.stub().returns(BPromise.resolve());
+        sinon.stub().returns(BPromise.resolve(rootRamlFile));
```
Comment (fschargorosdky) [inline tests/unit/apiPlatformService.test.js]: check that rootRamlFile has been added to the result

Comment (juancoen) [inline tests/unit/apiPlatformService.test.js]: it's being checked with should.deepEquals(api, newApi)

Comment (juancoen) [inline tests/unit/apiPlatformService.test.js]: done

File: tests/unit/apiPlatformService.test.js [hunk 3]
```diff
@@ -35,6 +36,20 @@ var newApi = {
...
+  audit: {
+    created: {
+      date: '2015-12-12 12:00'
+    },
+    updated: {
+      date: '2015-12-12 12:00'
+    }
+  }
+};
+var expectedAPI = _.clone(newApi);
```
Comment (fschargorosdky) [inline tests/unit/apiPlatformService.test.js]: use _.cloneDeep

**Comment by fschargorosdky** [comment]
Looks good to me

**Pull request merged**